gcp/appengine/frontend_handlers.py

```diff
@@ -77,6 +79,54 @@ def index():
   return render_template('index.html')
 
 
+@blueprint.route('/v2/')
+def index_v2():
+  return render_template('home.html')
+
+
+@blueprint.route('/v2/list')
+def list():
+  """Main page."""
+  # TODO: Can/should we do this as an internal query not a full on request to osv.dev?
+  page = request.args.get('page') if request.args.get('page') and request.args.get('page').isnumeric() else '1'
+  response = requests.get('https://osv.dev/backend/query?page=%s&search=&affected_only=true&ecosystem=' % page)
+  results = json.loads(response.content)
+
+  vulnerabilities = []
+  for item in results['items']:
+    vulnerabilities.append({
+      "id": item['id'],
+      "summary": item['summary'] if 'summary' in item else '',
+      "packages": item['affected'][0]['package']['ecosystem'],
+      "versions": item['affected'][0]['versions']
+    })
+
+  return render_template('list.html', vulnerabilities = vulnerabilities)
+
+
+@blueprint.route('/v2/vulnerability/<id>')
+def vulnerability(id):
+  """Vulnerability page."""
+  # TODO: Can/should we do this as an internal query not a full on request to osv.dev?
+  response = requests.get('https://osv.dev/backend/vulnerability?id=%s' % id)
+  item = json.loads(response.content)
+
+  vulnerability = {
```

Comment:
Can we just return the vulnerability structure as-is, without doing this field mapping here? The structure format is stable and based on https://ossf.github.io/osv-schema/.

Reply:
SG, shall we do the same for the vuln list? From a look at the API it [does look the same](https://osv.dev/docs/#operation/OSV_QueryAffected)